.github/pull_request_template.md

```diff
@@ -9,7 +9,7 @@
 <!-- Describe why you're making these changes if it's not already explained in a corresponding issue. -->
 <!-- If you're closing an existing issue with this pull request, use the keyword Closes #issue_number -->
 
-<!-- Do you have any suggestions about this PR template? Edit it here: https://github.com/renovatebot/renovatebot.github.io/edit/build/.github/pull_request_template.md -->
+<!-- Do you have any suggestions about this PR template? Edit it here: https://github.com/renovatebot/renovatebot.github.io/edit/main/.github/pull_request_template.md -->
```

Comment:
```suggestion
<!-- Do you have any suggestions about this PR template? Edit it here: https://github.com/renovatebot/renovatebot.github.io/edit/HEAD/.github/pull_request_template.md -->
```

Will this work?

Reply:
I end up at this page when I copy/paste your link:

![after-clicking-on-suggested-link](https://user-images.githubusercontent.com/34918129/146172914-571d511d-e0b0-48a1-9816-5deceb1857c8.png)

I think this _might_ work properly if you're a maintainer, who doesn't have a fork... 😉 Can you try it out yourself and see if it works for you?

Reply:
nope, won't work. So ignore 🤗